packages/admin-panel/src/VizBuilderApp/api/queries/useEntities.js

```diff
@@ -12,7 +12,7 @@ export const useEntities = search =>
     ['entities', search],
     async () => {
       const endpoint = stringifyQuery(undefined, `entities`, {
-        columns: JSON.stringify(['name', 'code']),
+        columns: JSON.stringify(['name', 'id']),
```

Comment:
Are these your changes?

Reply:
Ah, yes. I guess I could just have the three properties? @IgorNadj

Reply:
Oh yep, I thought this might have accidentally come in from a merge or something. If these are your changes then yeah I'd say just add instead of modify.